lib/easy.c

```diff
@@ -681,6 +681,12 @@ static CURLcode easy_perform(struct Curl_easy *data, bool events)
   /* Copy the MAXCONNECTS option to the multi handle */
   curl_multi_setopt(multi, CURLMOPT_MAXCONNECTS, data->set.maxconnects);
 
+#ifdef USE_HTTP2
+  /* Copy the STREAM_WINDOW_SIZE option to the multi handle */
+  curl_multi_setopt(multi, CURLMOPT_STREAM_WINDOW_SIZE,
+                    data->set.stream_window_size);
+#endif
```

Comment:
The documentation says an individual easy handle's stream size setting is ignored by the multi, not inherited. The user is expected to set CURLMOPT_STREAM_WINDOW_SIZE.

Reply:
This particular setopt is called within the `curl_easy_perform()` function for the "internal" multi handle so using the easy handle's value for this seems correct to follow the docs.